src/combobox/combobox.js

```diff
@@ -251,6 +251,13 @@ function Combobox<OptionT>(props: PropsT<OptionT>) {
     {overrides: popoverOverrides = {}, ...restPopoverProps},
   ] = getOverrides(overrides.Popover, Popover);
 
+  // React-focus-lock used in Popover used to skip non-tabbable elements (`tabIndex=-1`) elements for focus, we had ListBox with tabIndex to disable focus on
+  // the ListBox, but we can just disable autoFocus (as ListBox / ListItem should not be focusable) (and input is also not autoFocused)
+  const popoverProps = {
```

Comment:
yeah, we could do that, it is just my personal preference of not doing that. I think if we do that, we cannot override the autoFocus value using restPopoverProps. 
```
const restPopoverProps = { autoFocus: true };
<FocusLock autoFocus={false} {...restPopoverProps} ></FocusLock> 

// In this case, the autoFocus will not be overridden when user passes autoFocus to true for any reason.
```
I see other components (like select) is already doing that (your suggestion) I can do that too.

Reply:
But I think we should avoid setting autoFocus to true even if the user passes it as true from above, because it makes the component start behaving wierdly.

Select does the same thing, Let me do that.

Thanks for that advice.